autoload/lh/cpp/GotoFunctionImpl.vim

```diff
@@ -452,8 +452,10 @@ function! s:BuildFunctionSignature4impl(proto,className)
         \ . '('.implParamsStr . ')'
         \ . (proto.const ? ' const' : '')
         \ . (!empty(proto.throw) ? ' throw ('.join(proto.throw, ',').')' : '')
-        \ . "\n{\n}"
-  return res
+        \ . "{\n}"
```

Comment:
It should even be `'{}'`. The `\n` should be taken care of by the style associated to `{}`.
Note that the default style associated to `{}` is `\n{}` and not `\n{\n}`.

I'll have to see If I cannot return something like `{!cursorhere!}` and play with my placeholder system instead.